app/assets/stylesheets/alchemy/archive.scss

```diff
@@ -85,20 +86,57 @@
     text-overflow: ellipsis;
   }
 
-  &:hover {
+  &:hover,
+  &:focus-within {
     .picture_tool,
     .picture_tags {
       visibility: visible;
       opacity: 1;
     }
   }
+
+  &:hover,
+  &:focus-within,
+  &.active {
```

Comment:
why not that block in line 91?

Reply:
Because only the select checkbox should be visible on active, not the delete button and I didnt want to repeat the active style

Reply:
active gets toggled during clicking the select all button, while the hover and focus states only happen on mouseover of the thumbnail